### PR TITLE
protect against NPE

### PR DESCRIPTION
**Pull request by kuisathaverat**
possible NPE

### CODE REVIEW

**Comment by kuisathaverat** [comment]
@jglick In that case, Should we annotate it as NonNull?

**Comment by jglick** [comment]
> Should we annotate it as NonNull?

Yes please.

File: src/main/java/io/jenkins/plugins/artifact_manager_jclouds/JCloudsArtifactManager.java [hunk 1]
```diff
@@ -94,7 +95,7 @@ private Object readResolve() {
     }
 
     @Override
-    public void onLoad(Run<?, ?> build) {
+    public void onLoad(@NonNull Run<?, ?> build) {
```
Comment (jglick) [inline src/main/java/io/jenkins/plugins/artifact_manager_jclouds/JCloudsArtifactManager.java]: Better to add annotations in the defining method in core.

Comment (kuisathaverat) [reply src/main/java/io/jenkins/plugins/artifact_manager_jclouds/JCloudsArtifactManager.java]: done https://github.com/jenkinsci/jenkins/pull/3543

**Pull request merged**